hc/accounts/tests/test_profile.py

```diff
@@ -18,8 +18,11 @@ def test_it_sends_set_password_link(self):
         self.alice.profile.refresh_from_db()
         token = self.alice.profile.token
         ### Assert that the token is set
+        self.assertIsNotNone(token)
```

Comment:
since tokens are typically a string check with the length of the token instead of using the `self.assertIsNotNone` token